libifupdown/lifecycle.c

```diff
@@ -225,6 +225,10 @@ build_environment(char **envp[], const struct lif_execute_opts *opts, const stru
 			if (iface->is_bridge)
 				lif_environment_push(envp, "IF_BRIDGE_PORTS", (const char *) entry->data);
 		}
```

Comment:
This will already be forwarded to the dhcp helper script as the IF_DHCP_CLIENT environment variable.  It does not require extension of the rigid interfaces file syntax.

Reply:
Thank you for reviewing!

I don't quite understand. I don't see any mentions of IF_DHCP_CLIENT variable in the code.

Reply:
All of the stanza configuration is automatically reflected into environmental variables.  See https://github.com/ifupdown-ng/ifupdown-ng/blob/main/libifupdown/lifecycle.c#L197 for how the configuration variables are mapped to environmental variables.